xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -1162,17 +1162,32 @@ Use <xref:System.Text.Json.Serialization.ReferenceHandler.Preserve> to enable un
         <summary>Gets or sets a value that indicates whether nullability annotations should be respected during serialization and deserialization.</summary>
         <value>To be added.</value>
         <remarks>
-          <para>Nullability annotations are resolved from the properties, fields and constructor parameters</para>
-          <para>that are used by the serializer. This includes annotations stemming from attributes such as</para>
           <para>
-            <see cref="T:System.Diagnostics.CodeAnalysis.NotNullAttribute" />, <see cref="T:System.Diagnostics.CodeAnalysis.MaybeNullAttribute" />,</para>
+            Configures the serializer to throw an exception when trying to serialize a <see langword="null" /> value from a non-nullable property getter,
+            or when deserializing a <see langword="null" /> value into a non-nullable property setter or constructor parameter.
+            Nullability annotations are resolved from the properties, fields, and constructor parameters
+            that are used by the serializer. This includes annotations stemming from attributes such as
+            <see cref="T:System.Diagnostics.CodeAnalysis.NotNullAttribute" />, <see cref="T:System.Diagnostics.CodeAnalysis.MaybeNullAttribute" />,
+            <see cref="T:System.Diagnostics.CodeAnalysis.AllowNullAttribute" /> and <see cref="T:System.Diagnostics.CodeAnalysis.DisallowNullAttribute" />.
+          </para>
           <para>
-            <see cref="T:System.Diagnostics.CodeAnalysis.AllowNullAttribute" /> and <see cref="T:System.Diagnostics.CodeAnalysis.DisallowNullAttribute" />.</para>
-          <para>Due to restrictions in how nullable reference types are represented at run time,</para>
-          <para>this setting only governs nullability annotations of non-generic properties and fields.</para>
-          <para>It cannot be used to enforce nullability annotations of root-level types or generic parameters.</para>
-          <para>The default setting for this property can be toggled application-wide using the</para>
-          <para>"System.Text.Json.Serialization.RespectNullableAnnotationsDefault" feature switch.</para>
+            Due to restrictions in how nullable reference types are represented at run time,
+            this setting only governs nullability annotations of non-generic properties, fields, and constructor parameters.
+            It cannot be used to enforce nullability annotations of root-level types, collection elements, or generic parameters.
+            The default setting for this property can be toggled application-wide using the
+            "System.Text.Json.Serialization.RespectNullableAnnotationsDefault" feature switch.
```

Comment:
What's `"System.Text.Json.Serialization.RespectNullableAnnotationsDefault"`? I cannot see it anywhere in MS Docs.

Reply:
It's a feature switch, but there are no conceptual docs created for it yet.

Reply:
Similar style being followed here, in case we want to update that as well:

https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.isreflectionenabledbydefault?view=net-8.0#remarks